labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -37,7 +41,17 @@ object RecursionPatternMatchingExercise {
    * List(1,1,2,3,1,1) -> List(1,1), List(2), List(3), List(1,1)
    */
   def groupConsecutive[T](in: List[T]): List[List[T]] = {
-    error("fix me")
+    in match {
+      case Nil => Nil :: Nil
```

Comment:
Mmm, I'm remembering this now. 

I guess the real question is whether [] is a tail of [].
I agree that it probably isn't, so [] is a correct solution